### ISSUE TITLE
couldn't send messages anymore (Have no target devices in PEP!?)

### ISSUE THREAD

**Issue by r-j-s** [siacs/Conversations#2561]
For a few days i am unable to send messages to my jabber.de contacts via Conversations. Can anyone help me who has better knowledge of the Conversations debug output than me?

#### General information

* **Version:** 1.19.4 (both clients)
* **Device:** n.a.
* **Android Version:** Android 7.1.2 LineageOS
* **Server name:** mailbox.org -> jabber.de
* **Server software:** n.a.
* **Installed server modules:** n.a.
* **Conversations source:** PlayStore (gApps)

#### Debug output

Result from "adb -d logcat -v time -s conversations" when try to send message to a jabber.de contact:

07-23 13:31:42.918 D/conversations( 2445): app switched into foreground
07-23 13:31:46.687 D/conversations( 2445): AxolotlService (xxx@mailbox.org): Finding devices without session for xxx@jabber.de
07-23 13:31:46.687 W/conversations( 2445): AxolotlService (xxx@mailbox.org): Have no target devices in PEP!
07-23 13:31:46.688 D/conversations( 2445): AxolotlService (xxx@mailbox.org): Cache miss: f03c0bda-3dd2-4dec-81bf-83176d655e37
07-23 13:31:46.891 D/conversations( 2445): AxolotlService (xxx@mailbox.org): Generated message, caching: f03c0bda-3dd2-4dec-81bf-83176d655e37
07-23 13:31:46.895 D/conversations( 2445): AxolotlService (xxx@mailbox.org): Cache hit: f03c0bda-3dd2-4dec-81bf-83176d655e37
07-23 13:31:53.420 D/conversations( 2445): app switched into background

**Comment by iNPUTmice** [comment]
Can you receive messages?
Please try unencrypted messages.

**Comment by r-j-s** [comment]
Hello iNPUTmice,

thx for your fast reply. To your question: yes i can receive messages from my contact (encrypted with OMEMO as well as unencrypted)
Unencrypted messages are also not possible to send...

**Comment by iNPUTmice** [comment]
Please contact your server administrator.

Or I guess maybe the recipients servers. I think the dialback might be hanging.

**Comment by r-j-s** [comment]
I did this and i wait for a reply of mailbox.org.
One problematic thing for me is that i could not see if a message was sent or not. For me it looks like there is everything OK and my contact will never get my message...
I know there is a setting in the conversations setting category "privacy" / "read- and receivenotification to my contacts" (or similar, I just translated it), but this didn't work for me. And I know that my contact has checked this setting as well as myself.

So is it possible to implement more verbosity in conversations, to recognize things like a dialback hanging, etc.
